frontend/src/components/c-summary-charts.vue

```diff
@@ -153,7 +153,8 @@
     .summary-chart(
       v-for="(user, j) in getRepo(repo)",
       v-bind:style="isChartGroupWidgetMode && j === getRepo(repo).length - 1 ? {'marginBottom': 0} : {}",
-      v-bind:ref="'summary-chart-' + j"
+      v-bind:ref="'summary-chart-' + j",
+      v-bind:id="user.name === activeUser && user.repoName === activeRepo ? 'selected' : null"
```

Comment:
We can make the `id` more descriptive, for example, `'selectedChart'`.
```suggestion
      v-bind:id="user.name === activeUser && user.repoName === activeRepo ? 'selectedChart' : null"
```

Reply:
fixed this!